includes/apple-exporter/components/class-video.php

```diff
@@ -20,6 +20,26 @@
  */
 class Video extends Component {
 
+	/**
+	 * Check if the remote file's headers return HTTP 200
+	 *
+	 * @param \DOMElement $node The node to examine for matches.
+	 * @return boolean
+	 * @access protected
+	 */
+	protected static function remote_file_exists( $node ) {
```

Comment:
Let's just implement this at the root component level - this is a good enhancement